packages/apollo-server-plugin-response-cache/src/ApolloServerPluginResponseCache.ts

```diff
@@ -177,7 +160,10 @@ export default function plugin(
             requestContext.overallCachePolicy = value.cachePolicy;
             requestContext.metrics.responseCacheHit = true;
             age = Math.round((+new Date() - value.cacheTime) / 1000);
-            return { data: value.data };
+            return {
+              data: value.data,
+              extensions: value.extensions,
```

Comment:
I think we'd want to avoid adding `extensions: undefined` if the key was originally missing, even if they JSON-serialize the same.